### PR TITLE
Fixed back link issue on opening with empty referrer

### PR DESCRIPTION
**Pull request by vrrayz**
Back button on openings now list to list page as requested from this https://github.com/Joystream/pioneer/issues/2977

@traumschule @oleksanderkorn

### CODE REVIEW

**Review by traumschule — APPROVED** [review]
tested [here](https://github.com/Joystream/pioneer/pull/3798#pullrequestreview-1214906856)

File: packages/ui/src/common/components/page/PreviousPage.tsx [hunk 1]
```diff
@@ -8,14 +9,21 @@ import { Arrow } from '../icons'
...
+    history.goBack()
+  }
+}
+export const PreviousPage = ({ children, showOverflow, customLink }: PreviousPageProps) => {
   const history = useHistory()
   return (
     <PreviousPageBlock showOverflow={showOverflow}>
       <PreviousPageButtonContainer>
-        <PreviousPageButton onClick={() => history.goBack()} size="small" square>
+        <PreviousPageButton onClick={() => setPrevHistory(history, customLink!)} size="small" square>
```
Comment (oleksanderkorn) [inline packages/ui/src/common/components/page/PreviousPage.tsx:26]: I suggest to fix this by removing non null assertion and marking the customLink as nullable inside the method:
image

image

Comment (oleksanderkorn) [inline packages/ui/src/common/components/page/PreviousPage.tsx:26]:
```suggestion
        <PreviousPageButton onClick={() => setPrevHistory(history, customLink)} size="small" square>
```

**Review by oleksanderkorn — CHANGES_REQUESTED** [review]

**Review by thesan — APPROVED** [review]
Thanks that's a great solution :raised_hands:

**Pull request merged**